test/TestProjects/BodyAndPath-LowLevel/Generated/BodyAndPathClient.cs

```diff
@@ -150,5 +151,116 @@ private HttpMessage CreateCreateRequest(string itemName, RequestContent content,
             request.Content = content;
             return message;
         }
+
+        /// <summary> Resets products. </summary>
+        /// <param name="itemNameStream"> item name. </param>
+        /// <param name="content"> The content to send as the body of the request. </param>
+        /// <param name="contentType"> Upload file type. </param>
+        /// <param name="excluded"> Excluded connection Ids. </param>
+        /// <param name="options"> The request options. </param>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> CreateStreamAsync(string itemNameStream, RequestContent content, ContentType contentType, IEnumerable<string> excluded = null, RequestOptions options = null)
```

Comment:
I thought the request was for `RequestContent content, ContentType contentType` to be the first args, not just together?

Reply:
I may have misunderstood @KrzysztofCwalina here.  I think having RequestContent and ContentType come at the end of the parameter list (as the last required parameters) makes more sense in general.